source/extensions/filters/common/ext_authz/check_request_utils.cc

```diff
@@ -68,6 +68,15 @@ void CheckRequestUtils::setAttrContextPeer(envoy::service::auth::v2::AttributeCo
     }
   }
 
+  if (ssl != nullptr && !local) {
```

Comment:
It looks like that this statement is already being evaluated in the above block: https://github.com/envoyproxy/envoy/blob/e28ecab3a5b9ff304b5b12376d7155d7124ffde7/source/extensions/filters/common/ext_authz/check_request_utils.cc#L43

I think we can avoid unnecessary comparisons by moving this logic into here: https://github.com/envoyproxy/envoy/blob/e28ecab3a5b9ff304b5b12376d7155d7124ffde7/source/extensions/filters/common/ext_authz/check_request_utils.cc#L56

Reply:
Thanks! I should have spotted that before uploading!